test/unit/services/JobRoleServiceTests.ts

```diff
@@ -43,5 +63,29 @@ describe('getJobRoles', function () {
       }
     })
   })
-})
 
+  describe('getJobRoleById', function () {
```

Comment:
what about a does not exist test?

Reply:
Changed JobRoleService method to throw Does Not Exist error if 404 code caught & added new test to JobRoleServiceTests :)